bn-models/src/models/_base/base.ts

```diff
@@ -17,6 +17,6 @@ export interface Base {
   dataSource?: string; // hostname available?
   isDeleted?: boolean; // hide from user, show to Beth :)
   legacy?: any; // Other legacy data from original database import
-  changeLog?: any; // optional change history log
+  changeLog?: ChangeLog; // optional change history log
```

Comment:
I think we want this to be an array, so changeLog?: ChangeLog[];

Reply:
oh, yeah.  definitely!